src/components/PhotoFrame.tsx

```diff
@@ -645,7 +651,8 @@ const PhotoFrame = ({
                             };
 
                             const renderListItem = (
```

Comment:
this had the effect while scrolling the thumbnail are not loaded but it needs a debouncer too